clients/geth/specular/rollup/customlog/customlog.go

```diff
@@ -0,0 +1,29 @@
+package customlog
```

Comment:
nit: would the following structure work?
```
rollup/utils/log/log.go (package log)
rollup/utils/fmt/fmt.go (package fmt)
```
so you only need to change the imports rather than the invocations

Reply:
Yes, this is updated